SU-SWS/stanford_basic
# D8CORE-851 Fix margin for figures

# READY FOR REVIEW

# Summary
- Override browser styles for `<figure>` tags

# Need Review By (Date)
- eventually

# Urgency
- low

# Steps to Test
1. checkout this branch
1. add a large image in the WYSIWYG, add a caption to it and align it to the left
1. add a blob of text below or above it
1. save and verify the image left alignment is the same as the text left alignment

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
